post.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+    require __DIR__ . '/vendor/autoload.php';
+
+    use Twig\Environment;
+    use Twig\Loader\FilesystemLoader;
+
+    $loader = new FilesystemLoader(__DIR__ . '/templates');
+    $twig = new Environment($loader);
+
+    session_start();
+
+    ini_set('display_errors', 1);
+    error_reporting(E_ALL);
+    require __DIR__ . '/vendor/autoload.php';
+    define("IN_INDEX", 1);
+
+    include("config.inc.php");
+    include("functions.inc.php"); 
+    
+    $date = date('Y-m-d');
+    $currentDate = date("Y-m-d H:i:s");
+
+    if (isset($config) && is_array($config)) {
+
+        try {
+            $dbh = new PDO('mysql:host=' . $config['db_host'] . ';dbname=' . $config['db_name'] . ';charset=utf8mb4', $config['db_user'], $config['db_password']);
+            $dbh->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+        } catch (PDOException $e) {
+            print "Nie mozna polaczyc sie z baza danych: " . $e->getMessage();
+            exit();
+        }
+
+    } else {
+        exit("Nie znaleziono konfiguracji bazy danych.");
+    }
+
+    //print_R($_SERVER['REQUEST_URI']);
+    $postid = explode("/", $_SERVER['REQUEST_URI']);
+    print_R($postid[3]);
+
+    if(isset($_SESSION['id'])){
+    if (isset($_GET['show']) && intval($_GET['show']) > 0) {
+
+        $id = intval($_GET['show']);
+
+        $stmt = $dbh->prepare("SELECT * FROM a30_Post WHERE id = :id");
```

Comment:
Comments about naming of variables and structuring into functions apply to this code as well (and profile.php and other files as well).